tests_e2e/test_e2e.py

```diff
@@ -1,16 +1,15 @@
 import os
 
 from dotenv.main import find_dotenv, load_dotenv
-from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.firefox.webdriver import WebDriver
 from selenium.webdriver.firefox.options import Options
 from todo_app.app import create_app
 import pytest
 import pymongo
 from selenium import webdriver
 from threading import Thread
-from todo_app.data.mongo_items import MONGO_CLIENT
 
+os.environ['LOGIN_DISABLED'] = 'True'
```

Comment:
Bit safer to put it at the top of the app_with_temp_db fixture, after loading .env, so that it is definitely set correctly at the start of each e2e test. Just in case .env sets it to false or other tests/code modify it.